### PR TITLE
Blog post 2021-05-11 about good documentation and new author id

### PR DESCRIPTION
**Pull request by AliceStatler**

### CODE REVIEW

**Review by cgebken — COMMENTED** [review]
Great post :) 

Let me give you some background information for the suggested changes:

- We decided to use the "Why..." structure for titles that fit into that category as it gets better results, e.g. on Google :) (If you are fine with that, we will have to adapt the file name accordingly)

- 2021-05-13 would be Christi Himmelfahrt and I don't think you would like to work that da y:D

- We use one sentence per line as that's easier to maintain and review

- It felt weird to mention the article without linking to it in the first paragraph. I linked it here and removed the sentence in the second section :)

- Sometimes I was missing a "connection" between the different sentences... That's why I've added some sentences and reordered the sentences in the last section.

- Some sentences were pretty generalizing "Nobody really likes to call the support." or to strict "Good technical documentation only presents the information that is really needed to get the job done."

- The second section "What makes good technical documentation?" contains many different aspects. It was a bit hard to follow. Suggestion: Maybe you could use bullet points here?

- Second section: "However, technical documentation also needs to match the requirements of the target group." --> I was missing at least one example here :)

- Last question (just to be sure): The header image is okay to use / from e.g. Pixabay/Pexels?

**Comment by AliceStatler** [comment]
Thanks for the review! I really like your changes. Here are some of my thoughts:

>     * The second section "What makes good technical documentation?" contains many different aspects. It was a bit hard to follow. Suggestion: Maybe you could use bullet points here?
-> I've tried to use bullet points but wasn't happy with it. Instead, I've added two level three headings. Is it easier to follow now?

>     * Second section: "However, technical documentation also needs to match the requirements of the target group." --> I was missing at least one example here :)
-> Added a bit more text and restructured the second section.

>     * Last question (just to be sure): The header image is okay to use / from e.g. Pixabay/Pexels?
-> It is from pexels.
